src/elements/common/header/Header.js

```diff
@@ -13,6 +13,7 @@ import { VIEW_FOLDER, VIEW_SEARCH } from '../../../constants';
 import './Header.scss';
 
 type Props = {
+    canSearch: boolean,
```

Comment:
Unused?

Reply:
I think I removed it before I got chance to look at this comment.